book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  has_many :rentals
+  has_many :persons, through: rentals
+
+  def initialize(title, author, rentals)
+    @title = title
+    @author = author
+    @rentals = rentals
```

Comment:
- [ ] I think it will be better if your rental variable is initialized straight ahead as an empty array so that it will be clear immediately that we are trying to store list of rentals 👍🏽 

e.g ``@rentals = []``